client/src/pages/Login.js

```diff
@@ -2,12 +2,13 @@ import React from "react";
 import styled from "styled-components/macro";
 import Page from "../common/Page";
 
+const clientId = process.env.REACT_APP_GITHUB_CLIENT_ID;
```

Comment:
Consider renaming it (https://javascript.info/variables#uppercase-constants)
```suggestion
const CLIENT_ID = process.env.REACT_APP_GITHUB_CLIENT_ID;
```
or

```suggestion
const { REACT_APP_GITHUB_CLIENT_ID: CLIENT_ID } = process.env;
```